backend/accounts/models.py

```diff
@@ -9,6 +10,15 @@
 from phonenumber_field.modelfields import PhoneNumberField
 
 
+def get_user_file_name(instance, fname):
```

Comment:
The name `get_user_file_name` is a bit ambiguous. Will we plan to use only 1 filepath unique to each user, or is this specific to profile pics? Also, if it's getting a file path instead of the name itself, it may be better to name it `get_user_filepath`?